src/utils/addEventListener.js

```diff
@@ -1,13 +1,13 @@
-// @flow weak
+// @flow
 
 import addEventListener from 'dom-helpers/events/on';
 import removeEventListener from 'dom-helpers/events/off';
 
-export default function(node, event, handler, options) {
-  addEventListener(node, event, handler, options);
+export default function(node: Node, event: string, handler: EventHandler, capture?: boolean) {
+  addEventListener(node, event, handler, capture);
   return {
     remove() {
-      removeEventListener(node, event, handler, options);
+      removeEventListener(node, event, handler, capture);
```

Comment:
We are using dom-helpers, and I typed it for that specific code:
https://github.com/react-bootstrap/dom-helpers/blob/master/src/events/on.js